mythril_core/src/lib.rs

```diff
@@ -29,6 +29,7 @@ mod registers;
 pub mod time;
 pub mod tsc;
 pub mod vcpu;
+pub mod vga_logger;
```

Comment:
Hmm, I don't think I want another top level module for this. We should probably ultimately make a 'logger' module that has a 'serial_logger' and 'vga_logger' under it. But that may be too much for this PR. For now, just put it in the existing 'logger' file, I think.